Hello I have the hard time finding out the cause and fix due to this error.. When I manually trigger the postman the api of signup.. I can see it working properly however in the axios. When triggering the sigup with not duplicate data it will successfully saved to the database and generates the token  but When triggering the error of duplicate it won't proceed to the console.log do you have any idea the cause of the issue? 
